bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/candidate/CandidateObsComparator.java

```diff
@@ -34,7 +34,7 @@ public int compare(Obs o1, Obs o2) {
 			return direction * ((ret != 0) ? ret : -o1.compareTo(o2));
 		}
 		
-		return direction * attr.getComparator().compare(o1, o2);
+		return direction * attr.getComparator(variant.getSchedule().getMiddlePoint()).compare(o1, o2);
```

Comment:
This is used to sort candidate observations in the list, so yes I think middle of the night is the best we can do (unless we want a slider somewhere that lets us pick a reference time).